src/interface.jl

```diff
@@ -56,6 +56,21 @@ current `state` of the sampler.
 """
 function step end
 
+"""
+    step_warmup(rng, model, sampler[, state; kwargs...])
+
+Return a 2-tuple of the next sample and the next state of the MCMC `sampler` for `model`.
+
+When sampling using [`sample`](@ref), this takes the place of [`step`](@ref) in the first
```

Comment:
Maybe good to qualify the unexported functions?
```suggestion
When sampling using [`sample`](@ref), this takes the place of [`AbstractMCMC.step`](@ref) in the first
```